File: ejercicio6/circulo.php
<?php
class Circulo{

    public $radio; // Radio del círculo

    public function calcularArea(){ // Calcula el área del círculo

        return "Área: ".(($this->radio)*($this->radio))*3.14;

    }
}

$circulo = new Circulo();
$circulo->radio = 5;

echo $circulo->calcularArea();

?>

File: Ejercicio5/EJ51/src/Circulo.java

public class Circulo extends Figura {
	
	double radio;
	
	public Circulo(double radio) {
        this.radio = radio;    }
	
	@Override
    double calcularArea() { //Calcula el área del círculo
        return Math.PI * radio * radio;
    }
}
